include/device.h

```diff
@@ -6,14 +6,35 @@ namespace dronecore {
 
 class DeviceImpl;
 
+/**
+ * A Device represents a unit such as a drone or robot.
```

Comment:
I prefer "vehicle" to "unit". 
Is it worth saying here that while the API in theory can connect to any vehicle, it is designed for  UAS - copter, plane, vtols, and not for rovers, subs, antenna trackers?

Reply:
Changed in b1b42db.